src/pyasm/checkin/metadata.py

```diff
@@ -389,6 +395,9 @@ def get_initial_data(self):
 
 
     def get_metadata(self):
+        if not HAS_EXIF:
+            raise("Unable to import EXIF parser.")
```

Comment:
Why are not using the parserimporterror?

Reply:
Typo, will fix